src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -217,7 +217,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_allocate_win_request_put_get(MPIR_Win * w
         + MPIDI_OFI_align_iov_len(alloc_iovs * t_size)
         + MPIDI_OFI_IOVEC_ALIGN - 1;    /* in case iov_store[0] is not aligned as we want */
 
-    req = (MPIDI_OFI_win_request_t *) MPIR_Request_create(MPIR_REQUEST_KIND__RMA, 0);
+    req = MPIDI_OFI_win_request_create();
```

Comment:
I *really* like the win_request changes, but it's another one I think is not critical to the request pool implementation, right? I'd prefer again to split it out and review it separately (along with someone from Intel).